Mathlib/Topology/Instances/NNReal/Lemmas.lean

```diff
@@ -232,4 +232,50 @@ theorem tendsto_of_antitone {f : ℕ → ℝ≥0} (h_ant : Antitone f) :
 
 end Monotone
 
+/-- Given `f : ι → ℝ≥0` with bounded range and `n : ℕ`, we have `(iSup f) ^ n = iSup (f ^ n)`. -/
+theorem iSup_pow {ι : Sort*} [Nonempty ι] {f : ι → ℝ≥0} (hf_bdd : BddAbove (range f)) (n : ℕ) :
+    (⨆ i : ι, f i) ^ n = ⨆ i : ι, f i ^ n := by
```

Comment:
I don't see much value in docstrings that simply repeat the statement, especially when they don't use notation! Consider removing, maybe?